project/src/pages/room/room.tsx

```diff
@@ -1,226 +1,105 @@
                 <h2 className='property__inside-title'>What&apos;s inside</h2>
                 <ul className='property__inside-list'>
-                  <li className='property__inside-item'>
-                    Wi-Fi
-                  </li>
-                  <li className='property__inside-item'>
-                    Washing machine
-                  </li>
-                  <li className='property__inside-item'>
-                    Towels
-                  </li>
-                  <li className='property__inside-item'>
-                    Heating
-                  </li>
-                  <li className='property__inside-item'>
-                    Coffee machine
-                  </li>
-                  <li className='property__inside-item'>
-                    Baby seat
-                  </li>
-                  <li className='property__inside-item'>
-                    Kitchen
-                  </li>
-                  <li className='property__inside-item'>
-                    Dishwasher
-                  </li>
-                  <li className='property__inside-item'>
-                    Cabel TV
-                  </li>
-                  <li className='property__inside-item'>
-                    Fridge
-                  </li>
+                  {currentOffer.goods.map((item) => (
+                    <li key={item} className='property__inside-item'>
+                      {item}
+                    </li>
+                  ))}
                 </ul>
               </div>
               <div className='property__host'>
                 <h2 className='property__host-title'>Meet the host</h2>
-                <div className='property__host-user user'>
-                  <div className='property__avatar-wrapper property__avatar-wrapper--pro user__avatar-wrapper'>
-                    <img className='property__avatar user__avatar' src={'img/avatar-angelina.jpg'} width='74'
-                      height='74'
-                      alt='Host avatar'
-                    />
-                  </div>
-                  <span className='property__user-name'>
-                    Angelina
-                  </span>
-                  <span className='property__user-status'>
-                    Pro
-                  </span>
-                </div>
+                <HostUser user={currentOffer.host} />
                 <div className='property__description'>
                   <p className='property__text'>
-                    A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The
-                    building is green and from 18th century.
-                  </p>
-                  <p className='property__text'>
-                    An independent House, strategically located between Rembrand Square and National Opera, but where
-                    the bustle of the city comes to rest in this alley flowery and colorful.
+                    {currentOffer.description}
```

Comment:
Вкусовщина, но я обычно вытягиваю все поля из объекта чтобы писать. Но тут как тебе больше нравится
```suggestion
                    {description}
```